### PR TITLE
feat: endpoints for deleting sequences and getting the current value

### PR DESCRIPTION
**Pull request by tommydeboer**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [ ] Migration step added in case of breaking change
- [x] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [x] Clean commits
- [x] Added Feature/Fix to release notes

### CODE REVIEW

**Comment by tommydeboer** [comment]
[Endpoints: delete sequence + get sequence value](https://trello.com/c/KnYvPAFD/455-endpoints-delete-sequence-get-sequence-value)

File: docs/guide-pseudonymisation.md [hunk 1]
```diff
@@ -63,9 +63,19 @@ with a sequence there are two endpoints available:
 #### `GET <host>/plugin/metadata-manager/sequences`
 Returns the sequences present in the database.
 
+> Note: Keep in mind that the sequence identifiers contain number signs (#). These need to be URL
```
Comment (jelmerveen) [inline docs/guide-pseudonymisation.md:66]: why do we do this?

File: docs/guide-pseudonymisation.md [hunk 2]
```diff
@@ -63,9 +63,19 @@ with a sequence there are two endpoints available:
...
+
+#### `GET <host>/plugin/metadata-manager/sequences/<sequence_id>`
+Returns the current value of the sequence.
+
+#### `DElETE <host>/plugin/metadata-manager/sequences/<sequence_id>`
+Deletes a sequence in the database. Since sequences are not automatically deleted when an entity type is deleted,
+you can use this endpoint to clean up. This endpoint can also be used to "reset" a sequence: a new sequence
+starting at 1 will be created as soon as a new row is added.
+
 #### `POST <host>/plugin/metadata-manager/sequences/<sequence_id>?value=123`
```
Comment (jelmerveen) [inline docs/guide-pseudonymisation.md:77]: make the title reflect the correct endpoint if # is enforced, so e.g value=%23123

File: molgenis-data-postgresql/src/test/java/org/molgenis/data/postgresql/JdbcSequencesTest.java [hunk 3]
```diff
@@ -32,8 +33,50 @@ void testGetSequences() {
 
   @Test
   void testSetValue() {
-    jdbcSequences.setValue("abc", 123);
-    verify(jdbcOperations).queryForMap("SELECT setval(?, ?)", "abc", 123L);
+    when(jdbcOperations.queryForList("SELECT sequence_name FROM information_schema.sequences"))
+        .thenReturn(List.of(Map.of("sequence_name", "seq1")));
+    jdbcSequences.setValue("seq1", 123);
```
Comment (jelmerveen) [inline molgenis-data-postgresql/src/test/java/org/molgenis/data/postgresql/JdbcSequencesTest.java:38]: how come value does not have a # here?

File: docs/guide-pseudonymisation.md (hunk 1 above)
Comment (tommydeboer) [reply docs/guide-pseudonymisation.md:66]: PostgreSQL has a restriction on the length of table (and sequence) names, which is a maximum of 32 characters. We don't have this restriction on entity type identifiers and attribute names. So to make sure that every identifier fits in the database, we truncate the identifier and append a hash of the identifier to make sure that it's unique. In the case of the sequence identifiers, a hash of the entity type id + attribute name is used. The format we use is:

`<truncated_id>#<hash>`

Comment (jelmerveen) [reply docs/guide-pseudonymisation.md:66]: ah ok, maybe that is good to add then. max size 32 characters, else you need to url encode

**Review by jelmerveen — APPROVED** [review]

**Comment by tommydeboer** [comment]
[[PID] Assign PIDs to existing biobanks](https://trello.com/c/71jeJOVZ/483-pid-assign-pids-to-existing-biobanks)

**Pull request merged**